oam/install_scripts/mcs-primproc.service.in

```diff
@@ -6,8 +6,8 @@ After=mcs-controllernode.service
 
 [Service]
 Type=simple
-Environment="LD_PRELOAD=libjemalloc.so.1 libjemalloc.so.2"
-ExecStart=@ENGINE_BINDIR@/PrimProc
+ExecStart=/usr/bin/env bash -c "LD_PRELOAD=$(ldconfig -p | grep -m1 libjemalloc | awk '{print $1}') exec @ENGINE_BINDIR@/PrimProc"
+ExecStartPost=sleep 2
```

Comment:
Prevents "Could not connect to PMS1: Connection refused" errors in exemgr and ddlprocs journal"